UK/Data/Plugin/TopSky/TopSkyCPDLC.txt

```diff
@@ -1,8 +1,8 @@
 [DCL]
-DCL:*:SID:<adep2> PDC <number> . . . . . CLD <hour><min> <day><month3><year2> <adep2> <callsign><endheader> CLRD TO <ades> OFF <drwy> VIA  <sid> ALT <cfl> SQUAWK <assr>. <qnh>HPA. <nfreq>. REPORT ACFT TYPE AND STAND NUMBER ON <freq> WHEN FULLY READY.
+DCL:*:SID:<adep2> PDC <number> . . . . . CLD <hour><min> <day><month3><year2> <adep2> <callsign><endheader> CLRD TO <ades> OFF <drwy> VIA <sid> ALT <cfl> SQUAWK <assr>. <qnh>HPA. <nfreq>. REPORT ACFT TYPE AND STAND NUMBER ON <freq> WHEN FULLY READY.
 
 [FREETEXT]
-FREETEXT:R:SERVICE TERMINATED. MONITOR UNICOM 122.800
+FREETEXT:UNICOM:SERVICE TERMINATED. MONITOR UNICOM 122.800
```

Comment:
What's the effect on the aircraft side? As far as I understood the :R: provides them with an option to respond "ROGER" to the request. What happens now this has been changed? I imagine TopSky won't like the return of "UNICOM" in response to the sent message. Unable to test myself at the moment.